developers/weaviate/config-refs/env-vars.md

```diff
@@ -35,6 +35,7 @@ This page includes a comprehensive list of environment variables that can be use
 | <code>PERSISTENCE<wbr />_LSM<wbr />_ACCESS<wbr />_STRATEGY</code> | Function used to access disk data in virtual memory | `string` | `mmap` (default) or `pread` |
 | <code>DISK_USE<wbr />_WARNING<wbr />_PERCENTAGE</code> | If disk usage is higher than the given percentage a warning will be logged by all shards on the affected node's disk. See [Disk Pressure Warnings and Limits for details](/developers/weaviate/configuration/persistence.md#disk-pressure-warnings-and-limits). | `string - number` | `80` |
 | <code>DISK_USE<wbr />_READONLY<wbr />_PERCENTAGE</code> | If disk usage is higher than the given percentage all shards on the affected node will be marked as `READONLY`, meaning all future write requests will fail. See [Disk Pressure Warnings and Limits for details](/developers/weaviate/configuration/persistence.md#disk-pressure-warnings-and-limits). | `string - number` | `90` |
+| <code>WEAVIATE<wbr />_ASYNC<wbr />_INDEXING</code> | If set, Weaviate will perform indexing asynchronously to the object creation process. This can be useful for importing large amounts of data. (default: `false`) <br/><br/>Available in versions `1.22` and higher. | `string - true/false` | `false` |
```

Comment:
Have edited to `If set, Weaviate creates vector indexes asynchronously to the object creation process. This can be useful for importing large amounts of data.`